src/mlpack/core/math/random.hpp

```diff
@@ -48,6 +48,19 @@ inline void RandomSeed(const size_t seed)
   #endif
 }
 
+/**
+ * Set the random seed to a fixed number.
```

Comment:
Perhaps it's a good idea to add a reference to the PR here, ideally we use the function only for specific cases.